stampede.js

```diff
@@ -15,15 +15,22 @@ class Stampede {
     this.expiry = options.expiry;
     this.passphrase = options.passphrase;
     this.algo = options.algo || 'aes192';
-    this.adapter = options.adapter || this.adapter;
```

Comment:
no idea how `this.adapter` could have ever been anything other than undefined here?